### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by donaldli2020** [ProdigyErrorCodes/ProdigyErrorCodes#8] [Community Q&A] [answered]
Maybe the branch could be renamed because there are many errors, not just 418.

Post your opinions below!

**Comment by afkvido** [comment, marked as answer]
Hell yea.

 The main problem is the links, those would all die.

**Reply by donaldli2020** [reply]
That would be a problem because they don't redirect.

**Reply by afkvido** [reply]
yup. what should we name the new branch?

**Reply by afkvido** [reply]
also, we would need to update the free checkmark workflow

**Reply by donaldli2020** [reply]
> yup. what should we name the new branch?

Maybe "errors", but we should keep the branch and tell them to update their bookmarks.

**Reply by donaldli2020** [reply]
# Please update your bookmarks to https://www.example.com/

**Reply by afkvido** [reply]
that's a good idea

**Reply by donaldli2020** [reply]
I added the 418 branch and told people to update their bookmarks.

**Reply by afkvido** [reply]
ty :)
